### ISSUE TITLE
Classifying a new dataset

### ISSUE THREAD

**Issue by step8p** [SCLBD/DeepfakeBench#90]
Suppose I have a new dataset, including a set of fake and real videos. It is not clear to me 

1. During training, where and how I specify the portion for training/test
2. Suppose I want to use the whole dataset for test only, what should I do?

Thanks!

**Comment by YZY-stack** [comment]
Hi, thanks for your interest.

If you want to use your own dataset in DeepfakeBench, you need to create the **JSON file** for each dataset.

For example, you have dataset-1 (d1). You can do the following steps:

1. You need to clarify all (processed img) paths of your dataset in **preprocessing/rearrange.py file**. After running this rearrange file, it will organize all your data for training and testing in a single JSON file.

2. Next, define the label for each dataset within the **training/config/train_config.yaml** and **training/config/test_config.yaml**.

3. Finally, you can train and test your dataset like other datasets within DeepfakeBench. For example, run the following command: python3 training/train.py --train_dataset d1 --test_dataset Celeb-DF-v2

By following these steps, you can seamlessly incorporate your own dataset into DeepfakeBench and leverage its capabilities for training and testing deepfake detection models.

**Comment by step8p** [comment]
perfect, thanks!!
